src/components/bulkConcepts/component/MappingPreview.jsx

```diff
@@ -23,10 +23,6 @@ const MappingPreview = ({ title, body }) => {
             Header: 'Relationship',
             accessor: 'map_type',
           },
-          {
```

Comment:
How is this related to the mappings not being displayed?

Reply:
It is not, directly. Given this functionality was not working before and this was a minor change raised within the same ticket, I felt it could be included.
New ticket?

Reply:
Yes that requires a new ticket.